content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -36,8 +36,22 @@ port: 20250
 serializeImagePulls: false
 evictionHard:
     memory.available:  "200Mi"
+    nodefs.available:  "10%"
+    imagefs.available: "15%"
+    nodefs.inodesFree: "5%"
 ```
 
+In this example, the Kubelet is configured with the following settings:
+
+1. address: The Kubelet will serve on IP address 192.168.0.8.
+2. port: The Kubelet will serve on port 20250.
+3. serializeImagePulls: Image pulls will be done in parallel.
+4. Eviction Thresholds: The Kubelet will evict Pods under the following conditions:
+   - When the node's available memory drops below 200MiB.
+   - When the node's main filesystem's available space is less than 10%.
+   - When the image filesystem's available space is less than 15%.
+   - When more than 95% of the node's main filesystem's inodes are in use.
```

Comment:
This seems to say the same kind of thing as the following paragraph; do we need them both?

Reply:
Sorry my bad :sweat_smile:, raised this PR in a hurry.

Reply:
I removed the old text now.